crates/neon-runtime/src/napi/bindings/types.rs

```diff
@@ -0,0 +1,154 @@
+use std::ffi::c_void;
+
```

Comment:
I think this file should have a comment at the top explaining the manual process we use for extracting the type declarations from nodejs_sys and pasting them here.

Reply:
I added this to `bindings/mod.rs` instead because the approach is identical in `types.rs` and `functions.rs`.